packages/graphql/tests/integration/issues/557.int.test.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Driver } from "neo4j-driver";
+import { graphql } from "graphql";
+import { gql } from "apollo-server";
+import neo4j from "../neo4j";
+import { Neo4jGraphQL } from "../../../src/classes";
+
+describe("https://github.com/neo4j/graphql/issues/557 - Input Object type ArticleCreateInput must define one or more fields", () => {
+    let driver: Driver;
+    let bookmarks: string[];
+    const typeDefs = gql`
+        type User557 {
```

Comment:
Not too important, but I think it is a better practice using unique, pseudorandom values for our types in integration tests, not only to solve name collision (in this case, it would be extremely rare for it to happen) but also for consistency (i.e. all integration tests using the same tooling that ensures no-collision, instead of relying on naming conventions)

Reply:
I've literally just tried to do this now, and until we fix our inconsistent pluralization, this is just so broken (the whole camel-casing thing). GIven I clean up in these tests, let's leave it given that it works for now?